docs/LDAP.rst

```diff
@@ -572,6 +572,45 @@ Configuration reference
 This section is a complete description of the available configuration options. 
 It lists the options of both eXo configuration and PicketLink configuration.
 
+.. _Ref_properties:
+
+Properties configuration
+------------------
+
+The LDAP integration uses jobs to synchronize periodically modified data on LDAP into eXo internal database.
+The list of properties that you can use in ``exo.properties`` are:
+
+.. code-block:: jproperties
+
+    # Cron expression used to schedule the job that will import periodically data
+    # from external store
+    # (Default value = every ten minutes)
+    exo.idm.externalStore.import.cronExpression=0 */10 * ? * *
+    # Cron expression used to schedule the job that will delete periodically data
+    # from internal store that has been deleted from external store
+    # (Default value = every day at 23:59 PM)
+    exo.idm.externalStore.delete.cronExpression=0 59 23 ? * *
+    # Cron expression used to schedule the job that will process periodically data
+    # injected in queue
+    # (Default value = every minute)
+    exo.idm.externalStore.queue.processing.cronExpression=0 */1 * ? * *
+
+By default, the user data are updated when the user login. This can be disabled by modifying this property:
+
+.. code-block:: jproperties
+
+    # if true, update user data on login time and only when the information changes
+    # on external store (Default: true)
+    exo.idm.externalStore.update.onlogin=true
+
+The user data import tasks are made asynchronously, thus a queue is used to process those tasks and a maximum number of synchronization operation retries can be configured by changing the following property:
```

Comment:
```suggestion
The user data import tasks are made asynchronously, thus a queue is used to process those tasks and a maximum number of synchronization operations retrials can be configured by changing the following property:
```

Reply:
are you sure that it's not `retries` instead:
https://www.linguee.fr/anglais-francais/traduction/number+of+retries.html
VS
https://www.linguee.fr/anglais-francais/traduction/retrial.html

Reply:
AH ok I misunderstood the signification